service/gcs/prot/protocol.go

```diff
@@ -172,15 +172,6 @@ type GcsCapabilities struct {
 	SendInitialCreateMessage bool
 }
 
-// Version is the struct sent to identify the protocol version of the HCS-GCS
-// communication protocol.
-type Version struct {
```

Comment:
**nit:** Might as well keep the `Version` struct, and just remove the `Patch` and `Metadata` fields, since we'll need `Version` later anyway.

Reply:
I will add it back as `SchemaVersion` because I think its more clear to differentiate them. To may `Version` concepts.

Reply:
Sounds good to me!